tests/v2/actions/etcdsnapshot/verify.go

```diff
@@ -0,0 +1,73 @@
+package etcdsnapshot
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	steveV1 "github.com/rancher/shepherd/clients/rancher/v1"
+	"github.com/rancher/shepherd/extensions/clusters"
+	extdefault "github.com/rancher/shepherd/extensions/defaults"
+	shepherdsnapshot "github.com/rancher/shepherd/extensions/etcdsnapshot"
+	"github.com/sirupsen/logrus"
+	kwait "k8s.io/apimachinery/pkg/util/wait"
+)
+
+// VerifySnapshots verifies that all snapshots come to an active state, and the correct number of snapshots were taken
+// based on the number of nodes and snapshot type (s3 vs. local)
```

Comment:
Since you have these broken up in two lines, can you start the second line with `and the correct number...`? It looks a bit awkward currently with how it's broken up.